frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-local-volume-set-step/create-local-volume-set-step.tsx

```diff
@@ -88,10 +88,9 @@ export const CreateLocalVolumeSet: React.FC<CreateLocalVolumeSetProps> = ({
   React.useEffect(() => {
     const createLvd = async () => {
       try {
-        setLvdInProgress(true);
         await k8sGet(LocalVolumeDiscovery, DISCOVERY_CR_NAME, csv?.metadata?.namespace);
-      } catch (e) {
-        if (e?.response?.status === 404) {
+      } catch (loadError) {
+        if (loadError?.response?.code === 404) {
```

Comment:
I think something got changed in request, and it was failing . Updated the status code key